x-pack/plugins/uptime/public/components/common/charts/donut_chart.tsx

```diff
@@ -57,9 +57,11 @@ export const DonutChart = ({ height, down, up }: DonutChartProps) => {
               'Pie chart showing the current status. {down} of {total} monitors are down.',
             values: { down, total: up + down },
           })}
-          {...chartTheme}
         >
-          <Settings theme={themeOverrides} />
+          <Settings
+            theme={[themeOverrides, chartTheme.theme ?? {}]}
+            baseTheme={chartTheme.baseTheme}
+          />
```

Comment:
The chart theme was being spread into the `<Chart>` component instead of the `<Settings>` component.